types/bun/package.json

```diff
@@ -0,0 +1,40 @@
+{
+    "private": true,
+    "name": "@types/bun",
+    "version": "1.0.9999",
+    "projects": [
+        "https://bun.sh"
+    ],
+    "dependencies": {
+        "bun-types": "1.0.18"
```

Comment:
Just to be sure, is the intent that you'll send a version bump each time? Or would unpinned be better?

Reply:
Without pinning, it's conceivable that some package managers will install an older version of `bun-types` from cache. Perhaps ironically this is the kind of optimization `bun install` is likely to do. 

It's not a big hassle for us to bump the dependency version, and it's easy to automate for us. Also - changing this value in the package.json is a perfect way for us to trigger DT's publishing CI.

Reply:
Alrighty, just know that there may be some delay depending on reviewers, though you may be able to self merge until you're popular. (Of course I can't not look at things I'm pinged on so... that's one way to guarantee a review 😅)

Reply:
Ah I did think we'd be able to self-merge going forward. Let's still leave it as is and see if it becomes burdensome for either us or the DT maintainers. Curious what the cutoff for "popular" is - `bun-types` gets just over 100k weekly downloads: https://www.npmjs.com/package/bun-types

> Of course I can't not look at things I'm pinged on

You are the platonic ideal of a maintainer 🫶 Take some breaks 😅